repose-aggregator/core/core-lib/src/main/java/org/openrepose/powerfilter/intrafilterLogging/RequestLog.java

```diff
@@ -41,17 +44,25 @@
     String requestBody;
     HashMap<String, String> headers;
 
+    /**
+     * Constructor populates all of the fields necessary for logging.
```

Comment:
There are a few things perplexing about this class.  I usually don't see package-level visibility for class fields.  To me that implies that there is another class in that same package that may be populating these fields (especially considering there are no setter methods).  I looked.  There is not.  Either it was a mistake or the serialization code requires it for some reason.  In any case, that particular JavaDoc addresses that.  The next time I have to go through this code, I don't have to worry about what else might be populating the fields.  I will know that at some point someone decided to tightly couple this POJO with MutableHttpServletRequest and FilterContext for some reason, and this JavaDoc allows me to remember this instantly.